Tests/StoreKitUnitTests/StoreProductTests.swift

```diff
@@ -107,14 +107,17 @@ class StoreProductTests: StoreKitConfigTestCase {
         expect(storeProduct.price.description) == "4.99"
         expect(storeProduct.priceDecimalNumber).to(beCloseTo(4.99))
         expect(storeProduct.localizedPriceString) == "$4.99"
-        expect(storeProduct.isFamilyShareable) == true
         expect(storeProduct.localizedTitle) == "Monthly Free Trial"
         // open the StoreKit Config file as source code to see the expected value
         expect(storeProduct.subscriptionGroupIdentifier) == "7096FF06"
 
         expect(storeProduct.subscriptionPeriod?.unit) == .month
         expect(storeProduct.subscriptionPeriod?.value) == 1
 
+        if #available(iOS 14.0, macOS 11.0, tvOS 14.0, watchOS 8.0, *) {
+            expect(storeProduct.isFamilyShareable) == true
+        }
```

Comment:
we don't need to fix this one right now, but we may have messed up here - [`isFamilyShareable` is available on watchOS 7](https://developer.apple.com/documentation/storekit/skproduct/3564805-isfamilyshareable#), not sure why we limited to 8 in StoreProduct but it was probably an oversight

Reply:
Huh thanks! I thought I saw this was watchOS 8.0 on `StoreKit` itself. I'll fix it!